src/main/app/form/validation.ts

```diff
@@ -88,3 +88,9 @@ export const isInvalidHelpWithFeesRef: Validator = value => {
     }
   }
 };
+
+export const isFieldLetters: Validator = value => {
+  if (!(value as string).match(/^[a-zA-Z][a-zA-Z\s]*$/)) {
```

Comment:
I don't think you need the first `[a-zA-Z]` in this regex, do you?

```suggestion
  if (!(value as string).match(/^[a-zA-Z\s]*$/)) {
```

Reply:
you do to accept uppercase letters.
the reason why i didnt use this `/^[a-zA-Z\s]*$/` is because this allows users to enter space only and it accepts it as its not empty.

Reply:
Ah I see, to exclude only spaces and also only match lowercase, uppercase and then spaces would this work?

```suggestion
  if (!(value as string).match(/^[^\s][a-zA-Z\s]*$/)) {
```

Reply:
this will allow a number or special character in the beginning